lib/msal-common/src/utils/Constants.ts

```diff
@@ -38,7 +38,8 @@ export const Constants = {
     AUTHORIZATION_PENDING: "authorization_pending",
     NOT_DEFINED: "not_defined",
     EMPTY_STRING: "",
-    FORWARD_SLASH: "/"
+    FORWARD_SLASH: "/",
+    EMPTY_OBJECT_STRING: "{}"
```

Comment:
What if someone passes "{ }"? Would it be better to do regex matching? Or at least remove whitespace before comparison?

Reply:
+1 to regex matching

Reply:
Good point, I will make some changes.

Reply:
Could consider using `JSON.parse` here also.

Reply:
e.g. `Object.keys(JSON.parse(" {  } ")).length`

Reply:
done.